### PR TITLE
Add Twala Jellie Testnet

### PR DESCRIPTION
**Pull request by apquinit**

### CODE REVIEW

**Comment by apquinit** [comment]
@ligi For your review and approval, please. Thanks!

File: _data/icons/twala.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://QmTXJVhVKvVC7DQEnGKXvydvwpvVaUEBJrMHvsCr4nr1sK",
```
Comment (Jay4577) [inline _data/icons/twala.json:3]: _data/icons/twala.json

**Review by ligi — CHANGES_REQUESTED** [review]
where did you pin the CID? I cannot resolve

File: _data/icons/twala.json (hunk 1 above)
Comment (apquinit) [reply _data/icons/twala.json:3]: What is the issue?

**Comment by apquinit** [comment]
> where did you pin the CID? I cannot resolve

I've used Pinata to pin the file:
https://gateway.pinata.cloud/ipfs/QmTXJVhVKvVC7DQEnGKXvydvwpvVaUEBJrMHvsCr4nr1sK

**Comment by ligi** [comment]
does not load for me

**Comment by apquinit** [comment]
> does not load for me

https://ipfs.io/ipfs/QmTXJVhVKvVC7DQEnGKXvydvwpvVaUEBJrMHvsCr4nr1sK

Looks like it's working just fine. Can you double check?

**Comment by apquinit** [comment]
> does not load for me

The pinned image is publicly viewable

**Comment by apquinit** [comment]
Hello, any updates?

File: _data/icons/twala.json (hunk 1 above)
Comment (marijnh) [reply _data/icons/twala.json:3]: There's no issue, going by user activity, this is either someone who doesn't know what they are doing or a spam bot. Advise to ignore.

**Review by ligi — APPROVED** [review]

**Pull request merged**